hack/container_digest.sh

```diff
@@ -0,0 +1,4 @@
+export OPERATOR_IMAGE_PULLSPEC='quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-operator@sha256:896ff6c6afcdd6d9cb80945868a39242f9441cf2ea5d424fc0e058800628b860'
```

Comment:
is it going to be an auto-generated file?
if so, is it possible to add a comment like `#auto-generated - do not edit` ? (surely I lack some context on how nudging config works)

Reply:
Not auto generated, we can manually add things here.

Konflux will create PR similar to this one:
https://github.com/konflux-ci/olm-operator-konflux-sample/pull/45